cmd/gator/test/gatortest_test.go

```diff
@@ -90,9 +96,27 @@ func Test_formatOutput(t *testing.T) {
 	}
 	for _, tc := range testCases {
 		t.Run(tc.name, func(t *testing.T) {
-			output := formatOutput(tc.inputFormat, tc.input)
-			if diff := cmp.Diff(tc.expectedOutput, output); diff != "" {
-				t.Fatal(diff)
+			require.NotEmpty(t, formatOutput(tc.inputFormat, tc.input))
```

Comment:
Because we are testing user-visible output, this test should fail if the output changes -- user-visible changes should be intentional.

Reply:
that makes sense; will revert changes to this file then and rework it once we do remove `evaluationmeta`